packages/types/lib/connect/api.ts

```diff
@@ -0,0 +1,28 @@
+import type { Endpoint, ApiError } from '../api.js';
+
+export type PostConnectSessions = Endpoint<{
+    Method: 'POST';
+    Path: '/connect/sessions';
+    Body: {
+        end_user: {
+            id: string;
+            email: string;
+            display_name?: string | undefined;
+        };
+        organization?:
+            | {
+                  id: string;
+                  display_name?: string | undefined;
+              }
+            | undefined;
+        allowed_integrations?: string[] | undefined;
+        integrations_config_defaults?: Record<string, { connection_config: Record<string, unknown> }> | undefined;
+    };
+    Error: ApiError<'forbidden' | 'invalid_body' | 'invalid_query_params' | 'internal_error'>;
```

Comment:
Old stuff

Reply:
yes. It is fixed in the other PR https://github.com/NangoHQ/nango/pull/2776/files#diff-bb1a9436ed436eaa04424f761fb7e7983183af72d751f15bb824395481e4adacL21